include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -2397,8 +2397,6 @@ void
 __pattern_sort(_Tag, _ExecutionPolicy&&, _RandomAccessIterator __first, _RandomAccessIterator __last, _Compare __comp,
                _IsMoveConstructible) noexcept
 {
-    static_assert(__is_serial_tag_v<_Tag> || __is_parallel_forward_tag_v<_Tag>);
-
```

Comment:
If you prefer to keep the assertion, try adding "or _IsMoveConstructible is false" to the assertion conditions.

Reply:
Fixed